include/entropy.h

```diff
@@ -64,6 +75,25 @@ static inline int _impl_entropy_get_entropy(struct device *dev,
 	return api->get_entropy(dev, buffer, length);
 }
 
+/**
+ * @brief Fills a buffer with currently available entropy.
+ *        Callable from Interrupt Service Routines.
+ *
+ * @param dev Pointer to the entropy device.
+ * @param buffer Buffer to fill with entropy.
+ * @param length Buffer length.
+ * @retval bytes filled on success.
+ * @retval -ERRNO errno code on error.
+ */
+static inline int entropy_isr_get_entropy(struct device *dev, u8_t *buffer,
```

Comment:
How about we have _isr at the end of the function as the naming convention, I think that will be cleaner going forward (entropy_get_entropy_isr)

Reply:
Agreed, will change